src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -46,6 +48,13 @@ public void storeMessage(Message message) {
     datastore.put(messageEntity);
   }
 
+  /** Inserted Code **/
+  public int getTotalMessageCount(){
+    Query query = new Query("Message");
+    PreparedQuery results = datastore.prepare(query);
+    return results.countEntities(FetchOptions.Builder.withLimit(1000));
```

Comment:
Would this limit the number of entities returned to 1000, therefore giving an inaccurate count if there are more than 1000 entries in the database?